library/network_connections.py

```diff
@@ -13,6 +13,10 @@
 
 # pylint: disable=import-error, no-name-in-module
 from ansible.module_utils.basic import AnsibleModule
+# pylint: disable=import-error, no-name-in-module
+from ansible.module_utils.network_lsr import ethtool
+
```

Comment:
This newline should be after AnsibleModule and not here.

Reply:
I added it after AnsibleModule but I think it is also needed there because is separating that line from: 
```
# pylint: disable=import-error, no-name-in-module                               
from ansible.module_utils.network_lsr import MyError   
```

Reply:
You can actully just use one pylint comment for the full block and enable the check afterwards with
`# pylint: enable=import-error, no-name-in-module`

Reply:
Thanks, done!